demo/dlb-libdlb-demo/test.sh

```diff
@@ -3,8 +3,7 @@
 dlb_dev=$(ls /dev/dlb* | sed 's/\/dev\/dlb//' | head -1)
 echo '\n1. Directed Traffic test'
 echo '--------------------------'
-/usr/local/bin/dir_traffic -n 128 -d $dlb_dev
+/usr/local/bin/dir_traffic -n 8 -d $dlb_dev -w epoll
 echo '\n2. Load Balanced Traffic test'
 echo '-------------------------------'
-/usr/local/bin/ldb_traffic -n 128 -d $dlb_dev
-
+/usr/local/bin/ldb_traffic -n 8 -d $dlb_dev -w epoll
```

Comment:
nitpick

Reply:
???

Reply:
@bart0sh Sorry, if you didn't understand.! But it seems that you changed! :)